scene/resources/primitive_meshes.h

```diff
@@ -152,8 +167,10 @@ class BoxMesh : public PrimitiveMesh {
 	static void _bind_methods();
 	virtual void _create_mesh_array(Array &p_arr) const override;
 
+	virtual void _update_lightmap_size() override;
+
 public:
-	static void create_mesh_array(Array &p_arr, Vector3 size, int subdivide_w = 0, int subdivide_h = 0, int subdivide_d = 0);
+	static void create_mesh_array(Array &p_arr, Vector3 size, int subdivide_w = 0, int subdivide_h = 0, int subdivide_d = 0, bool p_add_uv2 = false, const float p_uv2_padding = 0.0);
```

Comment:
Should this default to a padding value of ``1.0`` like the others?

Reply:
Not sure, the default is to not add padding so the function doesn't change its current behavior. But I guess setting it to 1.0 in case just `p_add_uv2` is set to true...

Reply:
ah, I really just used the 0.0 default in one place now did I... weird :) I'll make it consistent :)